bin/get_and_export_ephemeris.sh

```diff
@@ -0,0 +1,20 @@
+# Get ephemeris files and export env variable to access them.
+# Output path is accepted as an input; otherwise, falls back
+# to ../.ephemeris.
+# This script does NOT alter any other bash script.
```

Comment:
What does that mean, why would it?

Reply:
That has to do with the current suggestion on how to get ephemeris in the [README](README.md), which includes the following statement:
```
echo 'export LALPULSAR_DATADIR=$VIRTUAL_ENV/share/lalpulsar' >> ${VIRTUAL_ENV}/bin/activate
```

This makes downloading ephemeris a once-in-a-lifetime process; however, if you opt to use `bin/get_and_export_ephemeris.sh`, you have to do it every single time you start the virtual environment, as otherwise there would be no `LALPULSAR_DATADIR` variable in the environment (and hence the `-nc` option in `wget` is there).